src/epcpm/importexport.py

```diff
@@ -212,6 +214,13 @@ def full_export(
         include_uuid_in_item=include_uuid_in_item,
     )
 
+    epcpm.staticmodbustoc.export(
+        c_path=paths.staticmodbus_c,
+        h_path=paths.staticmodbus_c.with_suffix(".h"),
+        staticmodbus_model=project.models.staticmodbus,
+        skip_sunspec=skip_sunspec,
```

Comment:
Maybe a separate PR would be good to expand the user-facing feature to be described as 'skip Modbus interfaces' if that's what it is going to be.  Depending on whether it is the excessive pile of functions involved in the SunSpec interface or the flash/ram usage, it may be that the static Modbus interface could be included.  If the functions are the issue, our static Modbus design does not have all those functions and therefor wouldn't cause the same problem, if it is the functions and not the memory usage...  Though, even if it is the memory usage, that might still be enough lower to be ok with static Modbus.  So, maybe at some point try timing a fresh build with each of no Modbus, SunSpec, and static Modbus.

Just `skip` like below or `skip_staticmodbus` and cascade it through the other file.

```suggestion
        skip=skip_sunspec,
```

Reply:
Changed `skip_sunspec` to `skip_output` where appropriate. Didn't use `skip_staticmodbus` because there were several situations where code applied to both skipping sunspec and staticmodbus. Addressed in 9d3fb7655e8af0a99e045fb31cb7c641dca644f9.